### DISCUSSION TITLE
some mint apps can not be executed using ulauncher

### DISCUSSION THREAD

**Discussion by RadinShirzad** [Ulauncher/Ulauncher#1107] [General]
Hi;
Ulauncher has been my application launcher since the very beginning of my Linux journey,
But here is an issue I have been dealing with since that beginning
I use Linux mint (20.3/21) and some of the mint applications specifically ones that need root privileges (mintsources,lightdm-settings,mintdrivers) can't be executed using Ulauncher

OS: Mint 20.3/21
DE/WM: cinnamon with muffin, awesomeWM

**Comment by friday** [comment]
I never ran into this. Can you try with GParted?

**Reply by RadinShirzad** [reply]
GParted launches just fine

**Reply by friday** [reply]
GParted also requires sudo, so it's probably the apps that are the issue. But I can't tell what they'd be doing wrong.

**Comment by friday** [comment]
Try enabling this (PolicyKit): https://forums.linuxmint.com/viewtopic.php?p=2145432#p2145432

**Reply by RadinShirzad** [reply]
these guys' problems seem to be different from mine,
I don't get that authentication pop-up to begin with(my authentication agent works fine)
though I'm pretty sure that the problem is from my side because the same problem is present in dmenu and awesomeWM's startmenu(cinnamon startmenu has no problem launching these apps).
I did run both Ulauncher and Dmenu from a terminal, and these are the outputs when running lightdm-settings:
when running lightdm-settings in Ulauncher: `Refusing to render service to dead parents.`
when running lightdm-settings in Dmenu: `lightdm-settings needs to be run as root `

**Reply by friday** [reply]
I don't know enough about these issues as I never had them myself and don't know how to reproduce, but there's a lot on Linux Mint Forums about polkit issues: https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fforums.linuxmint.com+can%27t+launch+sudo+apps&oq=site%3Ahttps%3A%2F%2Fforums.linuxmint.com+can%27t+launch+sudo+apps, so maybe one of those relate to your issue. You can possibly also try gtk-launch (ex `gtk-launch gparted.desktop`, but for the apps you are having problems with) to see if it happens there.

It's probably not a Ulauncher issue or anything we could even handle without adding a sudo prompt for all apps.

**Reply by RadinShirzad** [reply]
ok, thanks for your time☺️
